modules/css/button/stories.tsx

```diff
@@ -405,6 +405,20 @@ storiesOf('CSS/Buttons/Icon Button', module)
       >
         <SystemIcon icon={activityStreamIcon} />
       </button>
+      <h3>Square with Color</h3>
+      <button className="wdc-btn-icon-square" aria-label="Activity Stream">
+        <SystemIcon icon={activityStreamIcon} fill="berrySmoothie600"></SystemIcon>
+      </button>
+      <button className="wdc-btn-icon-square" aria-label="Activity Stream">
+        <SystemIcon
+          icon={activityStreamIcon}
+          accent="fruitPunch600"
+          color="berrySmoothie600"
+        ></SystemIcon>
+      </button>
+      <button className="wdc-btn-icon-square" aria-label="Activity Stream">
+        <SystemIcon icon={activityStreamIcon} backgroundHover="toothpaste600"></SystemIcon>
+      </button>
```

Comment:
Ah we appreciate you updating this but this package is in `maintenance` mode, so there's no need to update it, maybe @willklein think it's a good idea?

Reply:
I agree, let's leave off the changes to this file altogether. We should only update something if there is a "breaking" change and we want to prevent an inaccuracy. If it's something additive like this, we can skip it.

Reply:
Sounds good! I removed the changes from this file.